__tests__/app.test.js

```diff
@@ -293,4 +295,50 @@ describe("app", () => {
     });
   });
 
-})
+  describe("11 GET query /api/articles?sort_by=created_at&order=desc", () => {
+    test("status: 400 returns articles sorted by the default criteria of created_at column and descending order", () => {
+      return request(app)
+      .get("/api/articles?sort_by=created_at&order=desc")
+      .expect(200).then(({body}) => {
+        expect(body.articles).toBeSortedBy("created_at", {
+          descending: true
+        })
+      })
+    })
+    test("status: 400 returns articles sorted by the author criteria and ascending order", () => {
+      return request(app)
+      .get("/api/articles?sort_by=author&order=asc")
+      .expect(200).then(({body}) => {
+        expect(body.articles).toBeSortedBy("author", {
+          descending: false
+        })
+      })
+    })
+    test("status: 200 returns with an array of CATS topics only", () => {
+      return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200).then(({body}) => {
+        expect(body.articles).toHaveLength(1)
+        body.articles.forEach((article) => {
+          expect(article.topic).toEqual("cats")
+        })
+      })
+    })
+    test("status 404: responds with an error when passed an invalid topic", () => {
+      return request(app)
+      .get("/api/articles?topic=linguistics")
+      .expect(404).then(({body : {msg}}) =>{
+        expect(msg).toEqual("Bad Request: Invalid input data.")
+      })
+    })
+    test("status 404: responds with an error when passed an invalid sort by option", () => {
+      return request(app)
+      .get("/api/articles?sort_by=mood")
+      .expect(404).then(({body : {msg}}) =>{
+        expect(msg).toEqual("Bad Request: Invalid input data.")
+      })
+    })
+  })
+  })

```

Comment:
Id like to see some tests for the sad paths for the sort_by and order queries and a query with all 3 valid possible queries